OrbitCore/TracepointInfoManager.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINTINFOMANAGER_H
```

Comment:
Be consistent with the include guards of other files. This would be `ORBIT_CORE_TRACEPOINT_INFO_MANAGER_H_`.